client/src/Components/Graph.jsx

```diff
@@ -0,0 +1,45 @@
+import React from "react";
+import { LineChart, Line, ResponsiveContainer, XAxis, YAxis, Tooltip } from "recharts";
+
+
+export default function Graph( props) {
+
+    function toIndianRupees(sum){
+        return Number(sum).toString().replace(/\D/g, "").replace(/(\d+?)(?=(\d\d)+(\d)(?!\d))(\.\d+)?/g, "$1,");
+    }
+
+    return (
+        <>
+            <div className='graphHeader'>
+                <p className='graph-text'>After <span className="timeperiod"> <b>{props.investmentPeriod} years</b> </span>, you will have<br />
+                    <span> <h2 className='totalamount'> <b> ₹ {props.result && toIndianRupees(props.result.sipGrowthResult)}</b></h2> </span>
+                    That's <span> <b className='potentialcapitalgain'>₹ {props.result && toIndianRupees((props.result.sipGrowthResult- props.monthlyInvestment*props.investmentPeriod))}</b> </span> as potential capital gains <br /> on your investment of
```

Comment:
why doing these calculations here , why are these calculations are on backend server

props.result && toIndianRupees((props.result.sipGrowthResult- props.monthlyInvestment*props.investmentPeriod)

this should be calculated at backend , and passed as a key , and u can use that key here